R/texreg.R

```diff
@@ -1925,6 +1925,467 @@ print.texregTable <- function(x, ...) {
+                } else {
+                    stop(paste0("Model ", i, ": wrong number of custom coefficient names."))
+                }
+            } else if (class(custom.coef.names) == "character") {
+                if (length(custom.coef.names) == length(models[[i]]@coef.names)) {
+                    models[[i]]@coef.names <- custom.coef.names
+                } else {
+                    stop(paste0("Model ", i, ": wrong number of custom coefficient names."))
+                }
+            } else {
+                stop(paste("Custom coefficient names must be provided as a list of", "character vectors."))
+            }
+        }
+        
+        coef.names <- models[[i]]@coef.names
+        co.names <- append(co.names, coef.names)
+        coefs <- models[[i]]@coef
+        co <- append(co, coefs)
+        label <- rep(paste0("Model", i), length(models[[i]]@coef))
+        lab <- append(lab, label)
+        if (length(models[[i]]@se) > 0) {
+            s.e <- models[[i]]@se
+            se <- append(se, s.e)
+        } 
+        if (length(models[[i]]@pvalues) > 0) {
+            pvalues <- models[[i]]@pvalues
+            pv <- append(pv, pvalues)
+        }
+        if (length(models[[i]]@ci.low) > 0) {
+            ci.lower <- models[[i]]@ci.low
+            ci.low <- append(ci.low, ci.lower)
+        }
+        if (length(models[[i]]@ci.up) > 0) {
+            ci.upper <- models[[i]]@ci.up
+            ci.up <- append(ci.up, ci.upper)
+        }
+    }    
+    dataframe <- data.frame(cbind(co.names, co, lab))
+    
+    if (length(models[[i]]@se) > 0) {
+        dataframe <- cbind(dataframe, se)
+    } 
+    
+    if (length(models[[i]]@pvalues) > 0) {
+        dataframe <- cbind(dataframe, pv)
+    } 
+    
+    if (length(models[[i]]@ci.low) > 0) {
+        dataframe <- cbind(dataframe, ci.low)
+    } 
+    
+    if (length(models[[i]]@ci.up) > 0) {
+        dataframe <- cbind(dataframe, ci.up)
+    } 
+    dataframe$co <- as.numeric(as.character(dataframe$co))
+    
+    if (length(dataframe$se) > 0) {
+        dataframe$se <- as.numeric(as.character(dataframe$se))
+    } 
+    
+    if (length(dataframe$pv) > 0) {
+        dataframe$pv <- as.numeric(as.character(dataframe$pv))
+    } 
+    
+    if (length(dataframe$ci.low) > 0) {
+        dataframe$ci.low <- as.numeric(as.character(dataframe$ci.low))
+    } 
+    
+    if (length(dataframe$ci.up) > 0) {
+        dataframe$ci.up <- as.numeric(as.character(dataframe$ci.up))
+    } 
+    
+    # aggregate confidence intervals or SEs
+    if (isTRUE(use.se[i]) && length(models[[i]]@se) > 0) {
+        lower.inner <- dataframe$co - dataframe$se
+        upper.inner <- dataframe$co + dataframe$se
+        lower.outer <- dataframe$co - 2 * dataframe$se
+        upper.outer <- dataframe$co + 2 * dataframe$se
+    } else if (length(models[[i]]@ci.low) == 0 && length(models[[i]]@se) > 0) {
+        z.inner <- qnorm(1 - ((1 - 0.5) / 2))
```

Comment:
You are hard-coding a significance level of 0.50 for the inner confidence intervals. Is this desirable? The user should at least be able to switch this on or off, if not replace by some arbitrary user-defined inner significance level. I think adding a logical argument that enables the user to switch off inner CIs would be the best solution.

Reply:
This is from the old `plotreg`, if you want I can add a new argument, but I can also change it or get rid of it if you think it is not necessary in this `plotreg` version

Reply:
Maybe we should get rid of the inner CIs because they don't convey any additional information? I included them originally because I thought they looked cool. What do you think?

Reply:
Ok, I coded it the same way as `ci.level` and called it `ci.inner`. Does it work?